src/lib/openapi/spec/environment-project-schema.ts

```diff
@@ -8,22 +8,37 @@ export const environmentProjectSchema = {
     properties: {
         name: {
             type: 'string',
+            example: 'development',
+            description: 'The name of the environment',
         },
         type: {
             type: 'string',
+            example: 'production',
+            description:
+                'The type of environment. Typically either development (non-production) or production',
```

Comment:
A couple things:
1. We should link to environment types here (added in the suggestion).
2. Is this a "read" schema / is it returned by a GET call? If so, I don't think we need the sentence about what it probably is. (removed in suggestion)

```suggestion
                'The [type of environment](https://docs.getunleash.io/reference/environments#environment-types).',
```